docs/network-requirements.md

```diff
@@ -0,0 +1,16 @@
+# Network Requirements
+
+Antrea has a few network requirements to get started, ensure that your hosts and firewalls allow the necessary traffic based on your configuration.
+
+| Configuration              | Host(s)                          | ports/protocols   |
+| -------------------------- | -------------------------------- | ----------------- |
+| Antrea with VXLAN enabled  | All                              | UDP 4789          |
+| Antrea with Geneve enabled | All                              | UDP 6081          |
+| Antrea with STT enabled    | All                              | TCP/UDP 1607      |
+| Antrea with GRE enabled    | All                              | UDP 4754          |
+| Antrea with IPSec enabled  | All                              | UDP 500           |
+| All                        | kube-apiserver host              | TCP 443 or 6443\* |
+| All                        | antrea-controller APIServer host | TCP 10349         |
```

Comment:
Currently it's not determistic which "antrea-controller APIServer host" is. Maybe just say "All"?